src/command/render/pandoc.ts

```diff
@@ -1513,6 +1514,9 @@ async function resolveExtras(
       }
       fontdirs.add(font_cache);
     }
+    const srcDir = Deno.env.get("QUARTO_SRC_PATH") ||
+        join(quartoConfig.sharePath(), "../../src");
```

Comment:
We use `Deno.env.get("QUARTO_SRC_PATH") || join(quartoConfig.sharePath(), "../../src")` in a few different places in our codebase. I wonder if we should be adding a `srcPath()` to `quartoConfig`

Reply:
I've implemented `quartoConfig.srcPath()` in a separate commit on this PR. (I only found one other use?)